CONTRIBUTING.md

```diff
@@ -163,3 +173,180 @@ describe 'a dog', ->
 # Returns `undefined`.
 disablePackage: (name, options, callback) ->
 ```
+
+## Issue and Pull Request Labels
+
+This section lists the labels we use to help us track and manage issues and pull request. Most labels are used across the core `atom/atom` repository and all package repositories, but some are specific for `atom/atom`.
+
+[GitHub  search](https://help.github.com/articles/searching-issues/) makes it easy to use labels for finding groups of issues or pull requests you're interested in. For example, you might be interested in [open issues across `atom/atom` and all Atom-owned packages which are labeled as bugs, but still need to be reliably reproduced](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+user%3Aatom+label%3Abug+label%3Aneeds-reproduction) or perhaps [open pull requests in `atom/atom` which haven't been reviewed yet](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+repo%3Aatom%2Fatom+comments%3A0). To help you find issues and pull requests, each label is listed with search links for finding open items with that label in `atom/atom` only and also across all Atom repositories. We  encourage you to read about [other search filters](https://help.github.com/articles/searching-issues/) which will help you write more focused queries.
+
+The labels are loosely grouped by their purpose, but it's not required that every issue has a label from every group or that an issue can't have more than one label from the same group.
+
+Please open an issue if you have suggestions for new labels or if you notice some labels are missing on some repositories.
```

Comment:
Where should one open an Issue? Should they all just be opened on atom/atom?

Should we have a `label` label? :laughing: